docker/php/Dockerfile

```diff
@@ -0,0 +1,107 @@
+# To copy composer binary file
+FROM composer:1.8.0 as composer
+
+################################################################################
+
+FROM php:7.1-alpine as php7.1
+
+RUN apk add --no-cache \
+        autoconf \
+        g++\
+        git \
+        make \
+        zlib-dev \
+    && pecl install \
+        mongodb \
+    && apk add --no-cache \
+        libssh2-dev \
+    && pecl install \
+        ssh2-1.1.2
+
+RUN docker-php-ext-install \
+        zip \
+    && docker-php-ext-enable \
+        mongodb \
+        ssh2 \
+        zip
```

Comment:
iirc no need to enable an extension installed with `docker-php-ext-install` :wink: